test/presubmit-tests-gce-minikube.sh

```diff
@@ -1,87 +0,0 @@
-#!/bin/bash -ex
```

Comment:
This script is pretty generic - it just provisions a VM and runs arbitrary scripts on it. We might want to keep it (and just rename).

Reply:
I'm cleanup the files that are not being used. If this file is needed we could bring it back.